src/index.tsx

```diff
@@ -3,20 +3,17 @@ import App from './App';
 
 export {
   DateSelector,
-  Dropzone,
   SignaturePad,
   AutocompleteAddress,
   Input,
   MultiDropzone,
-  DownloadRing,
   IbanInput,
   CurrencyInput,
   BottomModal,
   RegularModal,
   BottomOrRegularModal,
   CardWithTopLeftIcon,
   CardWithLeftIcon,
-  CardWithTopIcon,
```

Comment:
Good stuff! ✨ The only one I would be careful with, would is probably `CardWithTopIcon`. Could you double check with the designers that this component is not used in the designs? Even if the component is not used in our codebases it doesn't necessarily mean it should be removed, it might just mean that people cooked up their own version of a component that already exists. In that case, refactoring the code to actually use it would be the more appropriate call.

Reply:
It definitely looks like something that might have been used in the designs:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/13664983/193021625-430b673f-5f40-484f-a35f-b5795373212c.png">

Reply:
There was some discussion about cards in general in terms of design (which lead to [this ticket](https://linear.app/feather-insurance/issue/STO-3862/split-card-ds-component-in-two) being closed) so it would be great to have the designers' opinion on this matter.

Reply:
Ayse did mention this design is being used, but when I examined the use case, it didn't seem like it could directly use this DS component. But I see your point, I will leave it here and we could update it to make it more reusable, then refactor the cards in website and app that fall into this category.